core/templates/css/oppia.css

```diff
@@ -1196,7 +1196,7 @@ textarea {
 
 .oppia-dashboard-aggregated-stats .stats-card {
   border-right: 1px solid #bbb;
-  margin: 10px 0;
+  margin: 10px 0 !important;
```

Comment:
using !important is not the preferred way to fix such issues. Can you use CSS specificity rules to achieve the required effect?

Reply:
@srijanreddy98  Can you please explain me a bit what you want?